apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx

```diff
@@ -23,7 +23,7 @@ export const WorkflowFloatingMenu: FC<IWorkflowFloatingMenuProps> = ({ className
   const { Component: DocsModal, setPath, toggle } = useDocsModal();
 
   const handleClick = (pathToSet: string) => () => {
-    setPath('framework/steps/' + pathToSet);
+    setPath('sdks/framework/typescript/steps/' + pathToSet);
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** This should ideally come from a constant -- especially one that lives with other "docs routes"‏